client/src/components/CreateBoardModal/CreateBoardModal.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Lifecycle from './Lifecycle';
+import { Button, Dialog, Classes, InputGroup, Label } from '@blueprintjs/core';
+
+class CreateBoardModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      autoFocus: true,
+      canEscapeKeyClose: true,
+      canOutsideClickClose: true,
+      enforceFocus: true,
+      isOpen: false,
+      usePortal: true
+    };
+    this.handleOpen = this.handleOpen.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+  }
+
+  handleOpen() {
```

Comment:
convert this into the arrow functions so that you can remove the bindings from the constructor.